src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/validation/CalculateMixingFractions.java

```diff
@@ -0,0 +1,139 @@
+package org.broadinstitute.hellbender.tools.walkers.mutect.validation;
+
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.vcf.*;
+import org.apache.commons.lang.mutable.MutableLong;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.VariantProgramGroup;
+import org.broadinstitute.hellbender.engine.*;
+
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.GATKProtectedVariantContextUtils;
+import org.broadinstitute.hellbender.utils.locusiterator.AlignmentStateMachine;
+import org.broadinstitute.hellbender.utils.pileup.ReadPileup;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+/**
+ * Given a VCF of known variants from multiple samples, calculate how much each sample
+ * contributes to a pooled BAM.
+ *
+ * In the CRSP sensitivity validation, we have a bam derived from a pool of 5, 10, or 20 samples
+ * and a vcf of all known variants in those samples.  The pooled bam is a simulated tumor and
+ * the vcf of individual variants is our truth data.  We annotate the truth data with the estimated
+ * allele fractions of each variant in the pooled bam in order to bin our results by "tumor" allele fraction.
+ * To estimate allele fractions we need to know the sample mixing fractions as an intermediate step.
+ *
+ * Example usage:
+ * java -jat gatk.jar CalculateMixingFractions -V input.vcf -O output.table
+ *
+ * Created by David Benjamin on 1/30/17.
+ */
+@CommandLineProgramProperties(
+        summary = "Calculate proportions of different samples in a pooled bam",
```

Comment:
Ditto...

summary and one-line summary should not be the same.

Also, perhaps you would like to add notes about who would be interested in this tool?  Is this really only ever used for doing an evaluation?